pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -187,20 +183,14 @@ func (m *kubeGenericRuntimeManager) generatePodSandboxLinuxConfig(pod *v1.Pod) (
 
 	if pod.Spec.SecurityContext != nil {
 		sc := pod.Spec.SecurityContext
-		if sc.RunAsUser != nil && runtime.GOOS != "windows" {
-			lc.SecurityContext.RunAsUser = &runtimeapi.Int64Value{Value: int64(*sc.RunAsUser)}
-		}
-		if sc.RunAsGroup != nil && runtime.GOOS != "windows" {
-			lc.SecurityContext.RunAsGroup = &runtimeapi.Int64Value{Value: int64(*sc.RunAsGroup)}
-		}
 		namespaceOptions, err := runtimeutil.NamespacesForPod(pod, m.runtimeHelper, m.runtimeClassManager)
```

Comment:
is this one linux only? Is there something we configure for Windows for user namespaces?

Reply:
Took a quick glance in ``NamespaceForPod``: it would seem so. First of all, it does have the ``Network`` field, which may be ``runtimeapi.NamespaceMode_NODE`` (host network) or ``runtimeapi.NamespaceMode_POD``, which is relevant for Windows as well. Not sure about the other fields though (``UsernsOptions`` may be relevant, since it's based on the ``Spec``'s ``RuntimeClassName``). I'd leave this here for the time being. But yeah, it's a bit strange that we need ``LinuxPodSandboxConfig`` for Windows.